lib/lookup.json

```
@@ -1,4 +1,11 @@
 {
+  "@babel/core": {
+    "maintainers": ["nicolo-ribaudo", "JLHwung", "liuxingbaoyu"],
+    "prefix": "v",
+    "yarn": true,
+    "scripts": ["build", "test"],
```

```suggestion
    "scripts": ["build", "test-only"],
```

I exposed a script that runs our tests without first linting.